upstream-sha256: 1a52c081e28185e7d283d4451654ebb3ebb48ac4c1f63a9f746e57543d0accec
actual-sha256: d9ba5834b063fe7a1942dc700dfdc18d1913c7deacf09f4cd467f0d39a3200e2
--- upstream.expected
+++ tolk.actual
@@ -8,25 +8,7 @@
     float val5 = (*(data1_128+(Lidx1+80)));
     float val6 = (*(data1_128+(Lidx1+96)));
     float val7 = (*(data1_128+Lidx1));
-    *(data0_16+Lidx1) = (val7+val1+val2+val3+val4+val5+val6+val0);
-  }
-}
----
-typedef float float4 __attribute__((aligned(16),ext_vector_type(4)));
-void E_4_4(float* restrict data0_16, float* restrict data1_16) {
-  for (int Lidx0 = 0; Lidx0 < 4; Lidx0++) {
-    int alu0 = (Lidx0<<2);
-    float4 val0 = (*((float4*)((data1_16+alu0))));
-    *((float4*)((data0_16+alu0))) = val0;
-  }
-}
----
-typedef float float4 __attribute__((aligned(16),ext_vector_type(4)));
-void E_4_4(float* restrict data0_16, float* restrict data1_16) {
-  for (int Lidx0 = 0; Lidx0 < 4; Lidx0++) {
-    int alu0 = (Lidx0<<2);
-    float4 val0 = (*((float4*)((data1_16+alu0))));
-    *((float4*)((data0_16+alu0))) = val0;
+    *(data0_16+Lidx1) = (0.0f+val7+val1+val2+val3+val4+val5+val6+val0);
   }
 }
 ---
